XP-Pen_Artist_16_Pro/index.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
This commit also adds `XP-Pen_Artist_156/XP-Pen_Artist_156.thumb.jpg`, which it shouldn't.